Readme.md

```diff
@@ -3,37 +3,28 @@
+The example demonstrates how to change a column value (Price) when another column (Percentage) is changed and vice versa.
 
+![Grid View - Changed Values](grid-batch-linked-values.png)
 
-```js
-var fieldName;
-function OnBatchEditStartEditing(s, e) {
-    fieldName = e.focusedColumn.fieldName;
-}
- 
+[ASPxClientGridView](https://docs.devexpress.com/AspNet/js-ASPxClientGridView) class implements the [batchEditApi](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.batchEditApi) property that exposes API to work with grid in batch mode on the client side.
 
-```
+When value editing is over, the [BatchEditEndEditing](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.BatchEditEndEditing) event is raised. In the event handler, you can get a cell value in the following ways:
 
+- [batchEditApi.GetCellValue](https://docs.devexpress.com/AspNet/js-ASPxClientGridViewBatchEditApi.GetCellValue(visibleIndex-columnFieldNameOrId)) method returns a value of a cell that is not in **editing mode**.
+- [rowValues](https://docs.devexpress.com/AspNet/js-ASPxClientGridViewBatchEditEndEditingEventArgs.rowValues) property returns a new entered value of a cell that is in **editing mode**.
```

Comment:
1. Please add an article ("The") before "rowValues".

I think "a new entered value" should be "the newly entered value".
However, is it necessary to say "newly entered"? For example, can we omit this phrase and say:

`- The [rowValues](https://docs.devexpress.com/AspNet/js-ASPxClientGridViewBatchEditEndEditingEventArgs.rowValues) property returns the value of a cell that is in **edit mode**.`

Reply:
I prefer to remain "the newly entered value", because the GetCellValue method can also return the value of a cell that is in **edit mode**, but it returns the old value.
```suggestion
- [rowValues](https://docs.devexpress.com/AspNet/js-ASPxClientGridViewBatchEditEndEditingEventArgs.rowValues) property returns the newly entered value of a cell that is in **editing mode**.
```

Reply:
Thank you for the suggestion. Can we change "editing mode" to "edit mode"?

```suggestion
- The [rowValues](https://docs.devexpress.com/AspNet/js-ASPxClientGridViewBatchEditEndEditingEventArgs.rowValues) property returns the new value of the specified cell that is in **edit mode**.
```